CHANGELOG.md

```diff
@@ -74,6 +74,9 @@ Unreleased changes template.
 Other changes:
 * (python_repository) Start honoring the `strip_prefix` field for `zstd` archives.
 * (pypi) {bzl:obj}`pip_parse.extra_hub_aliases` now works in WORKSPACE files.
+* (deps) rules_cc 0.0.16
+* (deps) bazel_skylib 1.7.0 (workspace; bzlmod already specifying that version)
+* (deps) protobuf 29.0-rc2 (workspace; bzlmod already specifying that version)
```

Comment:
We should add to breaking changes list that users will have to include extra deps loading. I previously thought that we can avoid that, but I don't think we can anymore, so `py_repositories_deps` call that you previously had in another PR may be the best choice now.

Reply:
There might be a solution for this. rules_cc 0.1.0 is an incompatible version that drops cc_proto_library from the defs file. Using it users of only rules_python, that don't need Protobuf don't need to do anything. Protobuf + WORKSPACE users, can either use 0.1.0 or the older version when one of their deps is still using cc_proto_library from defs.bzl.

Reply:
Fixed by using rules_cc 0.1.0